src/components/filters/DateFilter/utils/Translations/DateFilterTitle.ts

```diff
@@ -17,10 +19,21 @@ const formatAbsoluteDateRange = (
     from: Date | string,
     to: Date | string,
     translator: IDateTranslator,
-): string =>
-    moment(from).isSame(moment(to), "day")
-        ? formatAbsoluteDate(from, translator)
-        : `${formatAbsoluteDate(from, translator)}\u2013${formatAbsoluteDate(to, translator)}`;
+): string => {
+    let fromDate = from;
+    let toDate = to;
+    if (isString(fromDate)) {
+        fromDate = convertPlatformDateStringToDate(fromDate);
+    }
+
+    if (isString(toDate)) {
+        toDate = convertPlatformDateStringToDate(toDate);
+    }
+
+    return areSameDate(fromDate, toDate)
+        ? formatAbsoluteDate(fromDate, translator)
+        : `${formatAbsoluteDate(fromDate, translator)}\u2013${formatAbsoluteDate(toDate, translator)}`;
```

Comment:
it could be 
```
const fromDate = isString(from) ? convertPlatformDateStringToDate(from) : from;
const toDate = isString(do) ? convertPlatformDateStringToDate(to) : to;

if(areSameDate(fromDate, toDate)) { reutrn ...; }

return `${formatAbsoluteDate(fromDate, translator)}\u2013${formatAbsoluteDate(toDate, translator)}`;
```